README.md

```diff
@@ -52,12 +52,12 @@ Adjust config/environments files.
 ## Running
 
 ```
-bundle exec rake server
+bundle exec rackup -p 1791
```

Comment:
check README of geocoder-api : port should be optional and you need to add comment `#\ -p 1791` for default just before `run Api::Root` in server.ru